### PR TITLE
Update vendor of containers/storage and run codespell.

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None

```
[NO NEW TESTS NEEDED]

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
One question and all kinds of test unhappiness @rhatdan . 
Otherwise, LGTM

**Comment by saschagrunert** [comment]
This definitely needs the updated c/common PR: https://github.com/containers/common/pull/1077

File: Makefile [hunk 1]
```diff
@@ -258,7 +258,7 @@ test/version/version: version/version.go
 
 .PHONY: codespell
 codespell:
-	codespell -S bin,vendor,.git,go.sum,.cirrus.yml,"RELEASE_NOTES.md,*.xz,*.gz,*.ps1,*.tar,swagger.yaml,*.tgz,bin2img,*ico,*.png,*.1,*.5,copyimg,*.orig,apidoc.go" -L uint,iff,od,seeked,splitted,marge,ERRO,hist,ether -w
+	codespell -S bin,vendor,.git,go.sum,.cirrus.yml,"RELEASE_NOTES.md,*.xz,*.gz,*.ps1,*.tar,swagger.yaml,*.tgz,bin2img,*ico,*.png,*.1,*.5,copyimg,*.orig,apidoc.go" -L pullrequest,uint,iff,od,seeked,splitted,marge,erro,hist,ether -w
```
[the remark this answers is not in the document]
Comment (rhatdan) [reply Makefile:261]: Yes, this gets codespell to stop channing ERRO->ERROR,  For some reason capitalized it is ignored.

**Pull request closed without merging**